db/migrate/20190930142055_add_supports_subjects_flag_to_bookings_placement_dates.rb

```diff
@@ -0,0 +1,5 @@
+class AddSupportsSubjectsFlagToBookingsPlacementDates < ActiveRecord::Migration[5.2]
+  def change
+    add_column :bookings_placement_dates, :supports_subjects, :boolean
```

Comment:
What happens to existing placement_dates? Shouldn't this be getting a default set?

Reply:
Ah yeah, good catch. Should default to `true`